dali/test/python/test_pipeline.py

```diff
@@ -1707,3 +1707,12 @@ def get_output():
     out = get_output()[0].at(0)
     assert out[0] == -0.5 and out[1] == 1.25
 
+def test_return_constants():
+    pipe = dali.pipeline.Pipeline(1, 1, 0)
```

Comment:
Nitpick, no GPU is used in the test anyway:
```suggestion
    pipe = dali.pipeline.Pipeline(1, 1, None)
```